### ISSUE TITLE
[Bug] Build: 8860 Hammer++ Crash Exception Access Violation Error

### ISSUE THREAD

**Issue by HideorEscape** [ficool2/HammerPlusPlus-Issue-Tracker#187]
On the latest update of hammer++ version 8860 I got this error every time I tried to open my map or any map file:
Exception Code: 0xc0000005
Reason: Exception_Access_Violation

I'm using it for CSGO. Since I cannot open any map file I cannot use the mapping program.

**Comment by ficool2** [comment]
Send the crash dump

**Comment by HideorEscape** [comment]
It looks like It got fixed somehow because I opened my map in regular hammer and unselected some VIS Groups then I saved it. After this I was able to open my map and other vmf map files as well.

[bin.zip](https://github.com/ficool2/HammerPlusPlus-Issue-Tracker/files/8041708/bin.zip)

**Comment by ficool2** [comment]
During these crashes, were you 1) logged into Steam and 2) not using any controllers? All these crashes happened when trying to read input

**Comment by HideorEscape** [comment]
> During these crashes, were you 1) logged into Steam and 2) not using any controllers? All these crashes happened when trying to read input

Yeah, It is 1, I wasn't logged into Steam. I just tested it with and without Steam.
